scripts/configure-venue-access.ts

```diff
@@ -44,10 +26,7 @@ if (argv.length < 4) {
   usage();
 }
 
-const projectId = argv[0];
-const venueId = argv[1];
-const method = argv[2];
-const accessDetail = argv[3];
+const [projectId, venueId, method, accessDetail] = process.argv.slice(2);
```

Comment:
This line should replace line 24. Then the `if` check on line 25 should be checking each of these variables.

Basically should end up looking like this:
```typescript
const [projectId, venueId, method, accessDetail] = process.argv.slice(2);
if (!projectId || !venueId || !method || !accessDetail) {
  usage();
}
```

Reply:
Adding this, atm. 

There is another `usage()` call a couple of lines above this one.
```jsx
const argv = process.argv.slice(2);
if (argv.length < 4) {
  usage();
}
```

I'm removing this. Just leave a note to let you know.

Reply:
Yup, the intent of my above code suggestion was to replace all of that section, not just this line (review tools just didn't let me show that in any useful way)